zendesk_tickets_machine/boards/views.py

```diff
@@ -244,7 +244,18 @@ def get(self, request, slug):
                 )
 
                 result = zendesk_ticket.create(data)
-                each.zendesk_ticket_id = result['ticket']['id']
+                ticket = result.get('ticket')
+                if ticket:
+                    each.zendesk_ticket_id = ticket.get('id')
+                else:
+                    result_description = result.get('description')
+                    result_details = result.get('details')
+                    items = result_details.popitem()
+                    for error_item in items[1]:
+                        item_description = error_item.get('description')
+                        item_error = error_item.get('error')
+                        error_text = result_description + ': ' + item_description + ' (' + item_error + ')'
```

Comment:
In Python 3.6+, we use string interpolation. https://www.python.org/dev/peps/pep-0498/

Also, I think this line breaks PEP8.